specs/integration/config.js

```diff
@@ -0,0 +1,13 @@
+/* You can overwrite this config using the --config command-line option.
+See: https://github.com/MattiSG/Watai/wiki/Configuration
+E.g.: $ watai specs/integration/  --config '{"baseURL":"http://example.org"}'
+
+Note that you will have to set at least these config options
+in order to perform authenticated operations:
+
+$ --config '{"email":"david@larlet.fr","password":"testpassword","username":"David Larlet"}'
```

Comment:
Because each env will have its own config and I want it to let it be overwritten by the `~/.watai/config` file and that config file has a higher priority. It's a workaround to be able to use personal config or CLI. Is there a better option?